static/js/modules/filters.js

```diff
@@ -208,26 +208,11 @@ $('.field input[type="text"]').on('keypress', function(e) {
     }
 });
 
-function bindFileFilter() {
```

Comment:
Can we make this more general, like `bindDateFilter`? It will be used for contribution date, disbursement date, filing date, and who knows what else.